lib/src/base-styles/_nx-colors.scss

```diff
@@ -29,7 +29,7 @@
   --nx-color-text-disabled: var(--nx-color-interactive-border-disabled); // match disabled text and border
 
   --nx-color-link: var(--nx-swatch-blue-40);
-  --nx-color-link-focus: var(--nx-swatch-teal-45);
+  --nx-color-link-focus: var(--nx-swatch-blue-30);
```

Comment:
It's a little hard to tell, since sketch won't let me click the text in the focus example (I can only click the focus border), but I'm pretty sure the text color of a focused link is supposed to be the same as the text color of an ordinary link.  As a result this variable should be deprecated (it can't be removed for backwards compat reasons) and set to `--nx-color-link`.

Reply:
>  It's a little hard to tell, since sketch won't let me click the text in the focus example (I can only click the focus border), but I'm pretty sure the text color of a focused link is supposed to be the same as the text color of an ordinary link. As a result this variable should be deprecated (it can't be removed for backwards compat reasons) and set to --nx-color-link.

Thanks for catching this, I updated it here: c9bd26d419b0f5529a872cbd7938929541e26672